admin_guide/high_availability.adoc

```diff
@@ -302,44 +429,55 @@ endif::[]
 ifdef::openshift-origin[]
 ====
 ----
-$ oadm router ha-router-us-west --replicas=3 \
-    --selector="ha-router=geo-us-west" \
-    --labels="ha-router=geo-us-west" \
+$ oadm router ha-router-us-west --replicas=5 \
+    --selector="ha-svc-nodes=geo-us-west" \
+    --labels="ha-svc-nodes=geo-us-west" \
     --credentials="$KUBECONFIG" \
     --service-account=ipfailover
 ----
 ====
 endif::[]
 +
-[NOTE]
+Run the *geo-cache* service with a replica on each of the nodes. An example configuration
```

Comment:
I just view the file in github directly with link https://github.com/pecameron/openshift-docs/blob/c973e88256f7e58ed00915893a771c94946b8379/admin_guide/high_availability.adoc 
There are some format issues in the lines I commented above, not sure if it is github problem or doc itself.